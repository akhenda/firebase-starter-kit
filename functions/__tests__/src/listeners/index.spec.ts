/* eslint-disable jest/valid-describe-callback */
/* eslint-disable import/first */
// import { init } from '@tests/__utils__/setup';

// import setupEnvironment from '@tests/__utils__/setupEnv';
import authTests from './auth';
import firestoreTests from './firestore';
import serversTests from './servers';
import storageTests from './storage';

describe('Firebase Functions Unit Tests', () => {
  // beforeAll(beforeAllTests);

  // afterAll(afterAllTests);

  // beforeEach(beforeEachTest);

  authTests();
  firestoreTests();
  serversTests();
  storageTests();
});
