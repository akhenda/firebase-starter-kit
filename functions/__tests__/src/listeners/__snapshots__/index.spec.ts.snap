// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Firebase Functions Unit Tests Auth when a user signs up, a user profile is created in firestore 1`] = `
{
  "createdAt": Timestamp {
    "_nanoseconds": 919000000,
    "_seconds": 1672772162,
  },
  "email": "user-1T2E3S4T5U6S7E8R@example.com",
  "name": {
    "first": "Test",
    "last": "User",
  },
  "phone": "0720 123456",
  "photo": {
    "full": "https://placeimg.com/640/480/people",
    "l": "https://placeimg.com/640/480/people?size=720",
    "m": "https://placeimg.com/640/480/people?size=360",
    "s": "https://placeimg.com/640/480/people?size=200",
    "xl": "https://placeimg.com/640/480/people?size=1024",
    "xs": "https://placeimg.com/640/480/people?size=100",
    "xxl": "https://placeimg.com/640/480/people?size=1920",
  },
  "updatedAt": Timestamp {
    "_nanoseconds": 919000000,
    "_seconds": 1672772162,
  },
  "username": null,
}
`;
